mod_reforged/hooks/config/strings.nut

```diff
@@ -1454,12 +1461,12 @@ foreach (vanillaDesc in vanillaDescriptions)
 	}),
 	RF_Ghostlike = ::UPD.getDescription({
 		Fluff = "Blink and you\'ll miss me.",
-		Requirement = "Total armor weight up to 20"
+		Requirement = "Total armor weight of 20 or less"
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
 				"When the number of adjacent allies is greater than or equal to the number of adjacent enemies, you may ignore [Zone of Control|Concept.ZoneOfControl] for your next movement action."
-				"When wielding a melee weapon with up to 4 [Reach|Concept.Reach] when you end your movement adjacent to an enemy, the next attack against that enemy deals " + ::MSU.Text.colorPositive("25%") + " more damage and " + ::MSU.Text.colorPositive("+20%") + " damage ignoring armor. This bonus expires upon any action other than an attack."
+				"When wielding a melee weapon with up to 4 [Reach|Concept.Reach] when you end your movement adjacent to an enemy, the next attack against that enemy deals " + ::MSU.Text.colorPositive("25%") + " additional damage and " + ::MSU.Text.colorPositive("+20%") + " damage ignoring armor. This bonus expires upon using any action other than an attack."
```

Comment:
I don't think `upon using any action` is better than `upon any action`. I prefer the original as it flows better in language, `using` sounds too gamey and mechanical when referring to something like an `action` that a person `does`.

Reply:
How about `upon taking any action other than an attack`?

Reply:
Yes that sounds good.